.github/workflows/ci.yml

```diff
@@ -43,8 +43,53 @@ jobs:
         run: |
           mypy --config-file mypy.ini --ignore-missing-imports --strict --warn-unreachable snuba
 
+  snuba-image:
+    name: Build snuba CI image
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    outputs:
+      branch: ${{ steps.branch.outputs.branch }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Get branch name (merge)
+        id: branch
+        run: echo "::set-output name=branch::$(echo ${GITHUB_REF#refs/heads/} | tr / -)"
```

Comment:
Could you please add a comment as to what is the transformation that happens? Easier for the human passing along.

Reply:
Done